<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>digitalMini</class>
 <widget class="QWidget" name="digitalMini">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#digitalMini {border:1px solid rgb(0, 0, 0); }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="0" column="1">
      <widget class="QCheckBox" name="negadoChk">
       <property name="text">
        <string>Negar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nombreTxt"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="tipoInput">
       <property name="text">
        <string>Input Digital</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
